src/bci_build/package.py

```diff
@@ -458,6 +461,12 @@ def build_version(self) -> Optional[str]:
             return f"15.{int(self.os_version.value)}"
         return None
 
+    @property
+    def build_release(self) -> Optional[str]:
+        if self.os_version in CAN_BE_LATEST_OS_VERSION and self.min_build_release:
```

Comment:
Why this condition?

Reply:
we only want to set the %RELEASE% number on the newest SP, not on the older ones (otherwise they would have the same release number)